src/zenml/zen_server/template_execution/utils.py

```diff
@@ -322,10 +324,17 @@ def generate_dockerfile(
         pypi_requirements_string = " ".join(
             [f"'{r}'" for r in pypi_requirements]
         )
-        lines.append(
-            f"RUN pip install --default-timeout=60 --no-cache-dir "
-            f"{pypi_requirements_string}"
-        )
+
+        if handle_bool_env_var("ZENML_RUNNER_IMAGE_DISABLE_UV", False):
```

Comment:
On second thought, would you please add this to the constants module, so at least we have one central place where our env vars are present ? Otherwise it will be lost in space and time.